### DISCUSSION TITLE
Milvus 2.0 RNN feature?

### DISCUSSION THREAD

**Discussion by phosphorylation** [milvus-io/milvus#15600] [Q&A and General discussion] [answered]
just saw this as a planned feature in the **Milvus 2.0 vs. 1.x** graph but I searched all issues and discussions and none seems to mention it. 
Care to share some thoughts about it? Will you guys implement this?

**Comment by xiaofan-luan** [comment, marked as answer]
> just saw this as a planned feature in the **Milvus 2.0 vs. 1.x** graph but I searched all issues and discussions and none seems to mention it. Care to share some thoughts about it? Will you guys implement this?

We have plan to implement it but maybe later this year. If anyone free and wanna to take something that might be a choice

**Reply by xiaofan-luan** [reply]
I put the RNN on GSoc(Google summer of code)..hopefully someone would help on it
